initializr-generator/src/main/java/io/spring/initializr/generator/ServiceMetadata.java

```diff
@@ -0,0 +1,82 @@
+package io.spring.initializr.generator;
+
+import java.io.File;
+import java.util.*;
+
+public class ServiceMetadata {
+    public final static Map<String, List<String>> annotationMap = new HashMap<>();
+
+    public final static Map<String, List<String>> importMap = new HashMap<>();
+
+    public final static Map<String, Integer> portMap = new HashMap<>();
+
+    private final static List<ServiceLink> serviceLinks = new ArrayList<>();
+
+    static {
+        final String azureServiceBus = "azure-service-bus";
+        final String cloudConfigServer = "cloud-config-server";
+        final String cloudEurekaServer = "cloud-eureka-server";
+        final String cloudGateway = "cloud-gateway";
+        final String cloudHystrixDashboard = "cloud-hystrix-dashboard";
+
+        // Azure Service Bus
+        annotationMap.put(azureServiceBus, Arrays.asList("@SpringBootApplication", "@EnableDiscoveryClient"));
+        importMap.put(azureServiceBus, Arrays.asList(
+                "org.springframework.boot.autoconfigure.SpringBootApplication",
+                "org.springframework.cloud.client.discovery.EnableDiscoveryClient"));
+        portMap.put(azureServiceBus, 8081);
+        // Use relative path for services routed by gateway
+        serviceLinks.add(new ServiceLink(azureServiceBus, "/" + azureServiceBus + "/", false));
+
+        // Cloud Config Server
+        annotationMap.put(cloudConfigServer, Arrays.asList("@SpringBootApplication", "@EnableConfigServer"));
+        importMap.put(cloudConfigServer, Arrays.asList(
+                "org.springframework.boot.autoconfigure.SpringBootApplication",
+                "org.springframework.cloud.config.server.EnableConfigServer"));
+        portMap.put(cloudConfigServer, 8888);
+        serviceLinks.add(new ServiceLink(cloudConfigServer, "http://localhost:8888/application-native.yml", true));
```

Comment:
what happens if service is deployed in the cloud? would localhost still work?

Reply:
Will not work, opened issue #57